tests/rptest/tests/data_policy_test.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2021 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from ducktape.mark.resource import cluster
+from rptest.clients.kafka_cli_tools import KafkaCliTools
+from rptest.clients.types import TopicSpec
+
+import json
+
+from rptest.tests.redpanda_test import RedpandaTest
+
+
+class DataPolicyTest(RedpandaTest):
+    topics = (TopicSpec(partition_count=1, replication_factor=3), )
+
+    def __init__(self, test_context):
+        super(DataPolicyTest, self).__init__(test_context=test_context,
+                                             num_brokers=3)
+
+        self.kafka_tools = KafkaCliTools(self.redpanda)
+
+    def _get_data_policy(self, function_name, script_name):
+        return "function_name: {} script_name: {}".format(
+            function_name, script_name)
+
+    @cluster(num_nodes=3)
+    def test_set_data_policy(self):
```

Comment:
perhaps add a test that asserts that a brand new topic doesn't have a data policy to test the default case.